### ISSUE TITLE
t/transpose-validation-cc.t fails (with Business::CreditCard 0.34?)

### ISSUE THREAD

**Issue by eserte** [racke/Data-Transpose#25]
The t/transpose-validation-cc.t test may fail:
```
#   Failed test 'Discover card => Unknown (invalid)'
#   at t/transpose-validation-cc.t line 32.

#   Failed test 'Discover card => Unknown (invalid)'
#   at t/transpose-validation-cc.t line 32.
# Looks like you failed 2 tests of 116.
t/transpose-validation-cc.t ........ 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/116 subtests 
```

Statistical analysis shows that this only happens with Business::CreditCard 0.34, but not with earlier versions:
```
****************************************************************
Regression 'mod:Business::CreditCard'
****************************************************************
Name                   Theta          StdErr     T-stat
[0='const']           1.0000          0.0000    145279811326512608.00
[1='eq_0.34']        -1.0000          0.0000    -58701908433416728.00

R^2= 1.000, N= 49, K= 2
****************************************************************
```

**Comment by racke** [comment]
Yes, it looks like that we need to adjust these tests to the changes in the recent version of Business::CreditCard. Thanks for the investigations!

**Comment by racke** [comment]
Fix through #26, thanks a lot for the report @eserte !
